recipes/r-canopy/meta.yaml

```diff
@@ -1,39 +1,95 @@
+{% set version = '1.3.0' %}
+
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
 package:
   name: r-canopy
-  version: "1.2.0"
+  version: {{ version|replace("-", "_") }}
 
 source:
-  url: https://cran.r-project.org/src/contrib/Archive/Canopy/Canopy_1.2.0.tar.gz
-  md5: 8e5231f15c346d39e64f96c3e934ce8a
+  fn: Canopy_{{ version }}.tar.gz
+  url:
+    - https://cran.r-project.org/src/contrib/Canopy_{{ version }}.tar.gz
+    - https://cran.r-project.org/src/contrib/Archive/Canopy/Canopy_{{ version }}.tar.gz
+
+
+  sha256: 8d87184ee99efdcec58dc8199aacbed5398be88050832ec3d9c4c10fef85e924
+  # patches:
+   # List any patch files here
+   # - fix.patch
 
 build:
+  # If this is a new build for the same version, increment the build number.
   number: 0
+
+  # This is required to make R link correctly on Linux.
   rpaths:
     - lib/R/lib/
     - lib/
 
+
 requirements:
   build:
-    - r-base
+    - r-base >=3.4
     - r-ape
-    - r-spam
     - r-fields
     - r-pheatmap
     - r-scatterplot3d
 
   run:
-    - r-base
+    - r-base >=3.4
     - r-ape
-    - r-spam
     - r-fields
     - r-pheatmap
     - r-scatterplot3d
 
 test:
   commands:
-    - $R -e "library('Canopy')"
+    # You can put additional test commands to be run here.
+    - $R -e "library('Canopy')"  # [not win]
+    - "\"%R%\" -e \"library('Canopy')\""  # [win]
+
+  # You can also put a file called run_test.py, run_test.sh, or run_test.bat
```

Comment:
You can safely remove all unnecessary comments from the meta.yaml and the build.sh

Reply:
But of course not the `# [win]` directives, those are needed by conda